.github/workflows/push_pr.yml

```diff
@@ -21,13 +21,18 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
```

Comment:
Needed to use golangci-lint-action v3. Review [this](https://github.com/golangci/golangci-lint-action/pull/403) for details.